src/notation/internal/notationselectionrange.cpp

```diff
@@ -279,26 +301,32 @@ int NotationSelectionRange::sectionElementsMaxY(const NotationSelectionRange::Ra
         }
     }
 
-    if (maxY == INT_MAX) {
-        maxY = 0;
+    if (maxY == DBL_MAX) {
```

Comment:
I know that this is outdated now, but I still wanted to say that I disagree, because:
- Applying RealIsEqual to things that may be DBL_MAX seems dangerous, because it might lead to overflows and produce unexpected results (I've not reasoned about this much longer than 10 seconds but at least it isn't trivially clear that it's _not_ a problem so I assume it _is_ a problem)
- In this case, we do want an exact equality check, and don't need to worry about rounding errors. Rounding errors only occur when something is the result of arithmetic operations. But if maxY is the result of arithmetic operations, we are already sure anyway that it is not equal to DBL_MAX. So there is really no need to take rounding errors into account here. Doing so is wasteful for performance.